### ISSUE TITLE
Unable to Upload Firmware to ESP32-S3 Due to Deep Sleep Mode Interruption

### ISSUE THREAD

**Issue by ced-mos** [Xinyuan-LilyGO/LilyGo-EPD47#133]
**Issue Summary**: I am unable to upload new firmware to my ESP32-S3 using PlatformIO because the device enters deep sleep mode every 5 seconds with the current program, causing the connection to fail before the upload completes.

Steps Taken:

1. Reset Button: Tried pressing the reset button before the upload starts, but the upload never completes.

2. Erase Flash: Attempted to erase the flash using esptool.py with the command `esptool.py --port /dev/cu.usbmodem14101 --before default_reset erase_flash`:

Output:
```
esptool.py v4.7.0
Serial port /dev/cu.usbmodem14201
Connecting.............
Detecting chip type... ESP32-S3
Chip is ESP32-S3 (QFN56) (revision v0.1)
Features: WiFi, BLE, Embedded PSRAM 8MB (AP_3v3)
Crystal is 40MHz
MAC: xx:xx:xx:xx:xx
Uploading stub...

A fatal error occurred: Serial data stream stopped: Possible serial noise or corruption.
```

Additional Attempt: Tried using the `--before default_reset` option with no success.

Request: How can I successfully erase the flash or upload new firmware to my ESP32-S3 without the process being interrupted by the deep sleep mode?

Additional Information:

Device: ESP32-S3
Port: /dev/cu.usbmodem14201
Tool: PlatformIO, esptool.py

**Comment by lewisxhe** [comment]
You can't press the reset button before uploading. Have you read the FAQ?
![image](https://github.com/user-attachments/assets/4c088491-a4f5-4bb1-9941-2db7653fb00c)

**Comment by ced-mos** [comment]
I have to admit I missed this information because I was on the main branch of the repository. Here's a detailed explanation of my experience, solution and a recommendation for improvements:

First, thank you for your proposed solution in the FAQ. It worked for me, but not immediately.

For posterity, here are the issues and solutions I encountered:
1. The solution didn't work on the first attempt. I had to unplug the device several times, and I'm not sure what the exact issue was.
2. When uploading a new sketch via PlatformIO tasks, the device wasn't found. The solution was to delete the flash using the command:
   ```sh
   esptool.py --port /dev/cu.usbmodem14201 erase_flash
   ```
   After this, I was able to upload new sketches successfully.

Regarding why I missed the information:
- The main branch README states `T5-4.7-S3 version is [here]`, but the naming schema for the product is unclear. I've seen several descriptions like `T5-4.7-S3`, `LilyGo-EPD47`, `T5-4.7`, `T5 e-Paper`, and `Screen-4.7-S3`, and it's not clear if these refer to the same product.
- The README also mentions that `arduino-esp32` needs to be upgraded to version 2.0.3 to support ESP32-S3. I assumed this branch's README would also work with ESP32-S3, but the documentation isn't transparent about which information is relevant for which board.

Finally I will close this issue as my problem is resolved. However, to prevent additional support requests, I recommend:
1. Implementing a more consistent naming convention if possible.
2. Adding an FAQ step with a table to identify the relevant documentation based on board information, such as `Screen-4.7-S3`, the version and such.
